Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -1,12 +1,18 @@
-﻿namespace OJS.Services.Ui.Business.Validation.Implementations;
+﻿namespace OJS.Services.Ui.Business.Validations.Implementations.Contests;
 
 using System;
 using System.Linq;
 using OJS.Data.Models.Contests;
 using OJS.Services.Common.Models;
+using Infrastructure;
+using Validation;
 
 public class ContestValidationService : IContestValidationService
 {
+    private readonly IDatesService datesService;
+
+    public ContestValidationService(IDatesService datesService) => this.datesService = datesService;
```

Comment:
A wise man (Doncho:D) once told me it would be better if these validation services have no dependencies and are something like a pure class.

Reply:
As we discussed with @gogo4ds, it's okay to leave the dateService in this class. Please let us know if you have something in mind.

Reply:
@msmner Its not great if db calls for example are made here (or other expensive operations), hence dependencies like Data/Business services are not the greatest. Maybe this is what Doncho meant here, but if its something else, feel free to point it out.